tests/freqai/test_freqai_interface.py

```diff
@@ -271,6 +280,7 @@ def test_start_backtesting(mocker, freqai_conf, model, num_files, strat, caplog)
     strategy.freqai_info = freqai_conf.get("freqai", {})
     freqai = strategy.freqai
     freqai.live = False
+    freqai.activate_tensorboard = test_tb
```

Comment:
couldn't this be set directly in the freqai_config fixture? so it applies to all tests implicitly?

or are we not "fully" loading the config here - so this would be skipped?

Reply:
Good point, I think we can set it in the fixture.